charts/topolvm/templates/controller/deployment.yaml

```diff
@@ -74,8 +74,9 @@ spec:
           {{- with .Values.image.pullPolicy }}
           imagePullPolicy: {{ . }}
           {{- end }}
-          {{- with .Values.controller.resources }}
-          resources: {{ toYaml . | nindent 12 }}
+          {{- if .Values.resources.topolvm_controller }}
+          resources:
+            {{- toYaml .Values.resources.topolvm_controller | nindent 12 -}}
```

Comment:
We use `with` command for this `if` pattern. Could you alter this and others you changed?

Reply:
Ahh. That's right. Seems like I pushed some trial changes. Thanks. I re-pushed. 
Can you take a look again?